src/kvstore/Part.cpp

```diff
@@ -40,7 +40,8 @@ Part::Part(GraphSpaceID spaceId,
            std::shared_ptr<folly::IOThreadPoolExecutor> ioPool,
            std::shared_ptr<thread::GenericThreadPool> workers,
            wal::BufferFlusher* flusher,
-           std::shared_ptr<folly::Executor> handlers)
+           std::shared_ptr<folly::Executor> handlers,
+           std::shared_ptr<raftex::SnapshotManager> snapshotMan)
```

Comment:
maybe the parameter is too many, is there some way to compact it?

Reply:
Good suggestion!  After balance completed,  i will refactor the code here.